### PR TITLE
Add new cert endpoints

### PR DESCRIPTION
**Pull request by AlicxSSH**

### CODE REVIEW

File: privx_api/authorizer.py [hunk 1]
```diff
@@ -557,3 +560,28 @@ def search_certificates(
...
+            UrlEnum.AUTHORIZER.GET_CERTIFICATES_LIST,
+        )
+        return PrivXAPIResponse(response_status, HTTPStatus.OK, data)
+
+    def get_certificate_by_id(self, cert_id: str) -> PrivXAPIResponse:
+        """
+        Get Certificate by ID
+
+        Returns:
+            PrivXStreamResponse
```
Comment (RomanSemkin) [inline privx_api/authorizer.py:581]: Such func returns PrivXAPIResponse, not a PrivXStreamResponse

File: privx_api/authorizer.py [hunk 2]
```diff
@@ -557,3 +560,28 @@ def search_certificates(
             body=cert_params,
         )
         return PrivXAPIResponse(response_status, HTTPStatus.OK, data)
+
+    def get_certificates_list(self) -> PrivXAPIResponse:
+        """
+        Get all Certificates.
+
+        Returns:
+            PrivXStreamResponse
```
Comment (RomanSemkin) [inline privx_api/authorizer.py:569]: same here

File: privx_api/authorizer.py (hunk 1 above)
Comment (cuongssh) [reply privx_api/authorizer.py:578]: @AlicxSSH let's fix the remaining issues

Comment (AlicxSSH) [reply privx_api/authorizer.py:578]: i just need to fix the comments? the rest is ok?

**Review by cuongssh — APPROVED** [review]

**Comment by cuongssh** [comment]
Closed as PR-84 already contains these changes

**Pull request closed without merging**